STEERING-COMMITTEE.md

```diff
@@ -162,13 +118,158 @@ There are two ways to raise issues to the steering committee for decision:
    attention from the steering committee.
 
+## Composition
+
+The steering committee has 7 seats.  Five of the seats will be Contributing
+seats, open to any project contributor.  Two of the seats will be End-User
+seats, open only to active end-user community members.  See Candidate
+Eligibility for definitions of these categories.
+
+Steering Committee members serve for 2-year terms, staggered in order to
+preserve continuity.  Every year either two or three contributor seats are
+elected, and one end user seat is elected.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections are held annually. Six weeks or more before the
+election, the Steering Committee will appoint Election Officer(s) (see below).
+Four weeks or more before the election, the Election Officer(s) will issue a
+call for nominations, publish the list of voters, and open the call for
+exceptions.  One week before the election the call for nominations and
+exceptions will be closed.  The election will be open for voting not less than
+two weeks and not more than four.  The results of the election will be
+announced within one week of closing the election.  New Steering Committee
+members will take office on December 1 of each year.
+
+### Election Officer(s)
+
+Six weeks or more before the election, the Steering Committee will appoint
+between one and three Election Officer(s) to administer the election.  Elections
+Officers will be Knative community members in good standing who are eligible to
+vote, are not running for Steering in that election, and can make a public
+promise of impartiality.  They will be responsible for:
+
+* Making all announcements associated with the election
+* Preparing and distributing electronic ballots
+* Judging exception requests
+* Assisting candidates in preparing and sharing statements
+* Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote
+
+Anyone who has at least 50 contributions in the last 12 months is eligible to
+vote in the Steering election. Contributions are defined as opening PRs,
+reviewing and commenting on PRs, opening and commenting on issues, writing
+design docs, commenting on design docs, helping people on slack, participating
+in working groups, and other efforts that help advance Knative the project.
+
+[This
+dashboard]((https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year)
+shows only GitHub based contributions and does not capture all the contributions
+we value. We expect this metric not to capture everyone who should be eligible
+to vote. If a community member has had significant contributions over the past
+year but is not captured in the [stats.knative.dev
+dashboard](https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year),
+they will be able to submit an exception form to the Elections Officer(s) who
```

Comment:
the individual or a WG lead. I think WG leads would notice that an individual that is contributing that deserves to be included for voting, so the WG could be the one submitting the form.

Reply:
My preference would be for people to explain their own exception requests, unless there's a strong group-level preference for proxy exception requests